babel.config.js

```diff
@@ -1,6 +1,4 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-  plugins: [
-    "react-native-reanimated/plugin",
-  ],
+  plugins: ['react-native-reanimated/plugin'],
```

Comment:
what's the change ?

Reply:
it's a change done prettier all the other changes are that as well